app/assets/stylesheets/dropdown.css

```diff
@@ -0,0 +1,47 @@
+/* Dropdown Button */
+.dropbtn {
```

Comment:
Because this is used in every render, these styles can go in `application.css` file. That way you don't need to reference it manually.